clarity_ext/pdf.py

```diff
@@ -2,8 +2,8 @@
 
 
```

Comment:
This is another very small file. It not placing it in a common module we could place it in a subdirectory?